pandas/_libs/lib.pyx

```diff
@@ -65,24 +66,23 @@ cdef extern from "numpy/arrayobject.h":
 cdef extern from "src/parse_helper.h":
     int floatify(object, float64_t *result, int *maybe_int) except -1
 
-cimport pandas._libs.util as util
-from pandas._libs.util cimport is_nan, UINT64_MAX, INT64_MAX, INT64_MIN
+from pandas._libs cimport util
+from pandas._libs.util cimport INT64_MAX, INT64_MIN, UINT64_MAX, is_nan
 
 from pandas._libs.tslib import array_to_datetime
-from pandas._libs.tslibs.nattype cimport NPY_NAT, c_NaT as NaT
-from pandas._libs.tslibs.conversion cimport convert_to_tsobject
-from pandas._libs.tslibs.timedeltas cimport convert_to_timedelta64
-from pandas._libs.tslibs.timezones cimport get_timezone, tz_compare
 
 from pandas._libs.missing cimport (
+    C_NA,
     checknull,
-    isnaobj,
     is_null_datetime64,
-    is_null_timedelta64,
     is_null_period,
-    C_NA,
+    is_null_timedelta64,
+    isnaobj,
 )
-
+from pandas._libs.tslibs.conversion cimport convert_to_tsobject
+from pandas._libs.tslibs.nattype cimport NPY_NAT, c_NaT as NaT
+from pandas._libs.tslibs.timedeltas cimport convert_to_timedelta64
+from pandas._libs.tslibs.timezones cimport get_timezone, tz_compare
```

Comment:
putting tslibs imports before non-tslibs imports (within cython files) is intentional, can we update config to preserve this?

Reply:
There is an option to have groups in ```isort``` and then order them, so if we have two modules ```foo``` and ```bar```, and we want ```bar``` first, we can do:

(in setup.cfg)
```
[isort]
known_foo = pandas.foo
known_bar = pandas.bar
sections = bar,foo
```

we can achieve the desired behavior but it will be applied across all files (py, pyx, pxi).